app/modules.go

```diff
@@ -162,6 +162,7 @@ func orderBeginBlockers(allModuleNames []string) []string {
 	return ord.TotalOrdering()
 }
 
+// orderBeginBlockers returns EndBlockers (crisis, govtypes, staking) with no relative order.
```

Comment:
what does "no relative order" mean? Isn't the goal of the method to order them?
```suggestion
// OrderEndBlockers returns EndBlockers (crisis, govtypes, staking) with no relative order.
```